prime-router/metadata/HL7/catchall/hl7/resource/Provenance.yml

```diff
@@ -44,7 +44,7 @@ agent_1:
   vars:
     orgName: MSH.4.1
     altId: MSH.4.2
-    contryCode: MSH.17
```

Comment:
Can we add a test for this? (ignore me if you did, my browser is struggling to load all the files

Reply:
I'll use this comment as a placeholder to double check that all typos/extra updates have corresponding tests.

For file in specific, I have logged a ticket to remove it and a number of other mappings which are apparently unused (#15342). I think we'll be grooming that ticket on Monday so if it's determined it should stay then I think a ticket should be logged to add tests. This and a number of other resources are without integration tests, though it's possible those are all the unused ones.

Reply:
Going through all the typos found I did find that in [v251 mappings xtn.11](https://github.com/CDCgov/prime-reportstream/pull/15333/commits/a4ee5c9f3999c5ec2f162d79ae83e8ebfe9af6e4) had a typo and thus wasn't in the test data because it wouldn't have been mappable. [Have updated the v251 xtn test data](https://github.com/CDCgov/prime-reportstream/pull/15333/commits/9f2bc407a1240472c44a354b5d7b5fcd54923bc2).